api-report/attributor.api.md

```diff
@@ -35,34 +36,33 @@ export class Attributor implements IAttributor {
     tryGetAttributionInfo(key: number): AttributionInfo | undefined;
 }
 
-// @public (undocumented)
+// @internal (undocumented)
 export class AttributorSerializer implements IAttributorSerializer {
```

Comment:
does this actually need to be exported from the package?

Reply:
once I move the tests in sequence to this package, no. I'm open to using a kludge import path there (since it's just test code) and fixing this up now, or doing so in a follow-up before un-private-ing the package.

Reply:
IMO best left to a followup. It allows removal of a lot of the API surface that's marked internal right now, actually. Which is great. See [here](https://github.com/Abe27342/FluidFramework/pull/3) (also addresses your layering concern with sequence->attributor dep)